frontend/admin/webpack.base.js

```diff
@@ -36,8 +37,6 @@ module.exports = {
         ADMIN_APP_URL: JSON.stringify(process.env.ADMIN_APP_URL),
         ADMIN_APP_DIR: JSON.stringify(process.env.ADMIN_APP_DIR),
         BUILD_DATE: JSON.stringify(new Date()),
-        OAUTH_LOGOUT_URI: JSON.stringify(process.env.OAUTH_LOGOUT_URI),
-        OAUTH_POST_LOGOUT_REDIRECT: JSON.stringify(process.env.OAUTH_POST_LOGOUT_REDIRECT),
```

Comment:
These are runtime variables injected by Apache. They should not be in the Webpack config.